code/modules/mob/living/simple_animal/hostile/ipc_zombie.dm

```diff
@@ -7,9 +7,9 @@
 	blood_type = COLOR_OIL
 	health = 100
 	maxHealth = 100
-	melee_damage_lower = 20
+	melee_damage_lower = 15
 	melee_damage_upper = 20
-	armor_penetration = 40
+	armor_penetration = 10
```

Comment:
I'd probably make this 20 AP, because 15dmg 10 AP does only 5 damage to basic armour vests and gets totally stopped by upper tiers of armour

Reply:
maybe make it 15AP as 20 AP will make basic armour vests completely useless in blocking their damage.